src/main/java/org/opensearch/security/dlic/rest/validation/EndpointValidator.java

```diff
@@ -11,11 +11,7 @@
 import java.util.List;
 import java.util.Objects;
 
-import static java.util.function.Predicate.not;
-import static org.opensearch.security.dlic.rest.api.Responses.badRequestMessage;
-import static org.opensearch.security.dlic.rest.api.Responses.forbiddenMessage;
-import static org.opensearch.security.dlic.rest.api.Responses.notFoundMessage;
-import static org.opensearch.security.dlic.rest.support.Utils.withIOException;
+import static org.opensearch.security.dlic.rest.api.Responses.*;
```

Comment:
Looks like this star import crept in. There's an open issue to add enforcement to forbid * imports: https://github.com/opensearch-project/security/issues/3260

Reply:
yeap fixed looks like that. fixed manully